web/signature_manager.css

```diff
@@ -0,0 +1,598 @@
+/* Copyright 2022 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#addSignatureDialog {
+  --border-color: #8f8f9d;
+  --primary-color: #15141a;
+  --secondary-color: #5b5b66;
```

Comment:
These variables seem to use the same values as in:
https://github.com/mozilla/pdf.js/blob/90a5c37cb0131e8645bc036ec8e2baa1a1f8bca3/web/dialog.css#L20-L21

So, could we perhaps re-use those existing variables here?